server/game/cards/09.1-FtE/SceneOfTheCrime.js

```diff
@@ -0,0 +1,19 @@
+const ProvinceCard = require('../../provincecard.js');
+
+class SceneOfTheCrime extends ProvinceCard {
+    setupCardAbilities(ability) {
```

Comment:
Please use `AbilityDsl` instead of passing the `ability` parameter into `setupCardAbilities()`.